io.openems.edge.evcs.webasto.unite/src/io/openems/edge/evcs/webasto/unite/WebastoImpl.java

```diff
@@ -54,20 +54,15 @@
 		EdgeEventConstants.TOPIC_CYCLE_EXECUTE_WRITE //
 })
 public class WebastoImpl extends AbstractOpenemsModbusComponent
-		implements OpenemsComponent, Webasto, Evcs, ManagedEvcs, EventHandler {
+		implements
+			Webasto,
+			Evcs,
+			ManagedEvcs,
+			EventHandler,
+			OpenemsComponent {
```

Comment:
Hast du das Eclipse Formatter Plugin installiert?
Diese Änderung hat sich schon zum zweiten Mal einschlichen.

https://openems.github.io/openems.io/openems/latest/intellij.html

Reply:
Ja ich nutze den Eclipse Formatter. Der macht die Änderung, dass er die in eine eigene Zeile schiebt. Die Dokumentation von dem Plugin scheint dann noch nicht 100% richtig zu sein, da die Formatter Config noch anders eingestellt werden muss.